.github/workflows/linux_arm64.yml

```diff
@@ -113,21 +133,6 @@ jobs:
       #
```

Comment:
should system info printing be a part of docker image initialization?

Reply:
This system_info action is unified to be used on different OSes/runners, so I don't think that it should be a part of a docker pipeline (if that's what you mean)